DANS-KNAW/easy-update-solr-index
# Easy 1111 - dc_-velden direct uit EMD laten afleiden

fixes EASY-1111

 #### When applied it will
- Extract the information for the DC related search fields from the EMD datastream instead of the DC datastream. 
#### Where should the reviewer @DANS-KNAW/easy start?

nl.knaw.dans.easy.solr.SolrDocumentGenerator class
#### How should this be manually tested?

run the update script for one or more datasets and compare the solr index after and before the update. The DC related fields should be the same. If a dataset had an 'incomplete' DC datastream, then the missing DC fields values should be there after the update, because those are then extracted from the EMD.
